### ISSUE TITLE
OWASP vulnerabilities blacklisted, akka version is too old

### ISSUE THREAD

**Issue by alexmnyc** [akka/alpakka#2274]
Please upgrade Akka to the latest

https://github.com/akka/alpakka/blob/efde457ab791cfaf7f2b85942921e7ad350e071c/project/Dependencies.scala#L15

OWASP:

Please upgrade Akka to the latest otherwise dependency is flagged by OWASP checker https://github.com/jeremylong/dependency-check-gradle

http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2017-1000034

Akka versions <=2.4.16 and 2.5-M1 are vulnerable to a java deserialization attack in its Remoting component resulting in remote code execution in the context of the ActorSystem.
CWE-502 Deserialization of Untrusted Data

CVSSv2:
Base Score: HIGH (9.3)
Vector: /AV:N/AC:M/Au:N/C:C/I:C/A:C
CVSSv3:
Base Score: HIGH (8.1)
Vector: /AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H

References:
CONFIRM - http://doc.akka.io/docs/akka/2.4/security/2017-02-10-java-serialization.html
Vulnerable Software & Versions: (show all)

cpe:2.3:a:akka:akka:*:*:*:*:*:*:*:* versions up to (including) 2.4.16
...

**Comment by ennru** [comment]
Hi @alexmnyc 

Thank you for bringing this up, but I'm not sure where you got those Akka versions from. Did you analyse a very old project?

Alpakka hasn't used Akka 2.4 since long way back and we depend on Akka 2.5.31 (latest release in the 2.5 series) and Akka 2.6.4 (latest release in the 2.6 series).

**Comment by alexmnyc** [comment]
No. Was using latest akka 2.6.4 with the latest 2.0.0 release. Somehow it flags it still

**Comment by raboof** [comment]
Can you share how we can reproduce the OWASP tool flagging this? Which Alpakka dependency were you including?

I tried it with:

```
buildscript {
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath 'org.owasp:dependency-check-gradle:5.2.1'
    }
}

configurations {
  compile
}

dependencies {
  compile 'com.lightbend.akka:akka-stream-alpakka-file_2.12:2.0.0-RC2'
  compile 'com.typesafe.akka:akka-stream_2.12:2.5.31'
}

apply plugin: 'org.owasp.dependencycheck'
```

and then `gradle dependencyCheckAnalyze --info` does not seem to report any errors, but I'm not particularly familiar with Gradle, so I might be doing something wrong.

**Comment by alexmnyc** [comment]
@raboof  sorry about the late response. that looks correct.
```
./gradlew dependencyCheckAnalyze
open build/reports/dependency-check-report.html 
```

**Comment by raboof** [comment]
That gives an empty listing for me though?

![DeepinScreenshot_select-area_20200421114058](https://user-images.githubusercontent.com/131856/79851006-042bd980-83c5-11ea-8d6c-e94c82c393ac.png)

**Comment by raboof** [comment]
wait, 'dependencies scanned: 0' doesn't look right either. Could you help me reproduce?
